lib/tools/adb-commands.js

```diff
@@ -390,8 +391,13 @@ methods.isWifiOn = async function () {
 /*
  * on: true (to turn on) or false (to turn off)
  */
-methods.setWifiState = async function (on) {
-  await this.shell(['settings', 'put', 'global', 'wifi_on', on ? 1 : 0]);
+methods.setWifiState = async function (on, isSimulator = false) {
+  if (isSimulator) {
```

Comment:
@mykola-mokhnach could this _var_ name be changed to `isEmulator` since android are emulators not simulators